### DISCUSSION TITLE
Welcome to Lenovo-ThinkPad-T440s Discussions!

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T440s#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sxqib** [comment]
hey, I love your work! I was wondering that if your Lenovo T440s EFI will work on Lenovo x240? it has almost the same specs.

**Comment by MultimediaLucario** [comment]
If your Lenovo ThinkPad X240 has an i5-4300U CPU and not a i5-4200U or i3 then I think it would work perfectly.

**Reply by sxqib** [reply]
thanks for your reply! yes it has a i5-4300U CPU. I tried on monterey and it worked flawlessly!

**Comment by imcake** [comment]
hey, my t440s has i5 4210U with an inside nvidia graphics card， use your efi will got screen display problem， how can I shield the nvidia graphics card, thanks.

**Reply by imcake** [reply]
And it stuck in this screen

![微信图片_20221116100248](https://user-images.githubusercontent.com/3276093/202065581-929d6e0a-8696-4e7f-8502-58efd4ffd2c7.jpg)

**Reply by sxqib** [reply]
patch the display

**Reply by MultimediaLucario** [reply]
The EFI works better on Intel GPUs but I will get right to patching an EFI that is optimized for i5-4210U.

**Reply by MultimediaLucario** [reply]
Try this EFI with the MacBookAir6,2 SMBIOS: https://www.mediafire.com/file/k0cx1igi0fcviub/EFI_%2528ThinkPads_with_i5-4210U%2529.7z/file 

Let me know if it works. 😎

**Reply by imcake** [reply]
Thanks @sxqib and @MultimediaLucario , I try to add '-wegnoegpu‘ in boot-arg and enable the CSM in BIOS, the problem never happen again.

**Comment by marco9566** [comment]
Thank u so much for your work, I'm French and I have a Lenovo t440s
